src/main/java/com/airsquared/blobsaver/Main.java

```diff
@@ -50,60 +61,150 @@ public static void main(String[] args) {
         try {
             Class.forName("javafx.application.Application");
             if (PlatformUtil.isMac() || PlatformUtil.isWindows() || PlatformUtil.isLinux()) {
-                if (appPrefs.getBoolean("Start background immediately", false) && PlatformUtil.isMac()) {
-                    PlatformImpl.setTaskbarApplication(false);
-                }
-                JavaFxApplication.launchit(args);
+                /*if (appPrefs.getBoolean("Start background immediately", false) && PlatformUtil.isMac()) {
```

Comment:
Could you please remove the unnecessary comment?

Reply:
Done.